src/utils/table.ts

```diff
@@ -4,17 +4,19 @@ import { DEFAULT_TABLE_COLUMNS, DEFAULT_TABLE_ROWS } from "settings";
 
 import { Table } from "types/Table";
 
+const generateUniqueTableReference = (title: string) => `${title}-${uuid()}`;
+
 export const generateTable: (props: {
-  tableNumber: number;
+  tableNumber?: number;
   columns?: number;
   rows?: number;
 }) => Table = ({
   tableNumber,
   columns = DEFAULT_TABLE_COLUMNS,
   rows = DEFAULT_TABLE_ROWS,
 }) => {
-  const title = `Table ${tableNumber}`;
-  const reference = `${title}-${uuid()}`;
+  const title = tableNumber ? `Table ${tableNumber}` : 'New table';
```

Comment:
Should this be just `New table` or could it be something like `New table`, `New table (2)`, `New table (3)` as seen in other software?


[Q] Can it be quick and easy to do something like 

```suggestion
  const title = tableNumber ? `Table ${tableNumber}` : `New Table (${someShortGeneratedUniqueToken})`;
```

Reply:
won't do, there's no way to guarantee that table name is unique unless we're using a `uuid`-like indicator, which would ruin UX ( seeing something like `New table XXXX-XXXX-XXXX` isn't really helpful ) and adding a lot of if/else checks to ensure that the next added table's name is exactly `X+1` might lead to other problems
discussed in slack